airgun/views/job_invocation.py

```diff
@@ -54,6 +54,10 @@ class target_hosts_and_inputs(WizardStepView):
         ansible_roles_list = TextInput(id='ansible_roles_list')
         power_action = OUIASelect('OUIA-Generated-Select-single-34')
 
+        targetting_type = Select('OUIA-Generated-Select-single-2')
```

Comment:
I suggest using a locator parameter like this to generally target the correct element.

```suggestion
        targetting_type = Select(locator='//div[button[@aria-haspopup="listbox"]]')
```